components/fxa-client/ios/FxAClient/FxAccount.swift

```diff
@@ -67,9 +67,9 @@ class FxAccount: RustFxAccount {
         }
     }
 
-    override func fetchDevices() throws -> [Device] {
+    override func fetchDevices(forceRefresh: Bool = false) throws -> [Device] {
         return try notifyAuthErrors {
-            try super.fetchDevices()
+            try super.fetchDevices(forceRefresh: forceRefresh)
```

Comment:
It pains me that we changed the name of this arg in the swift API, but 👍 for keeping it consistent with `getProfile` 😄

Reply:
Yeah, I'm curious why iOS uses the fetchDevices/forceRefresh naming while android uses getDevices/ignoreCache. I almost didn't catch the iOS changes because of the difference. A part of me wants to rename the iOS functions but I also want to avoid causing breaking changes and possibly disregarding a sound design decision.

Reply:
if renaming is sound then, IMHO we should clean those up at some point and report it in breaking changes

Reply:
/cc @eoger